lib/utils/verify-signatures.js

```diff
@@ -0,0 +1,460 @@
+const cacache = require('cacache')
```

Comment:
We don't need any of the cacache stuff in here, `npm-registry-fetch` already does this under the hood.  What we really need is for the registry to return caching headers for the `-/npm/v1/keys` response.